src/main/java/Duke.java

```diff
@@ -1,10 +1,116 @@
+import java.io.*;
+import java.util.*;
+
 public class Duke {
+    public static void saveToFile(ArrayList<Task> taskItems){
```

Comment:
There should be a space before the open braces as specified in Egyptian style brackets